### ISSUE TITLE
Onboarding Suggestion

### ISSUE THREAD

**Issue by chostudio** [ryrotella/clippyBoard#1]
On onboarding, when clicking "grant" to allow accessibility, if it is possible to prompt within system settings to have a pop-up to add Clippy board (instead of having to click the plus button and search for it), that might be helpful.

**Comment by ryrotella** [comment]
Hey! Good suggestion. I looked into it. MacOS does have an API (AXIsProcessTrustedWithOptions) that can show a    
  system prompt and pre-register the app in the Accessibility list automatically. Unfortunately, it doesn't work for
  sandboxed apps, which this one is.

For now, I want to keep the app sandboxed, in the event I put this app in the App Store. But if that changes, I'll implement this! But based on your comment, I did update the UI to reflect what someone has to do in order to manually add. Thank you!
